src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/types.js

```diff
@@ -35,10 +36,21 @@ export type ItemActionConfig = {|
 
 export type ItemActionsProvider = (item: ?Object) => Array<ItemActionConfig>;
 
+export type IconSkin = 'dark';
+
+export type DateTimeSkin = 'light';
+
+export type AdapterOptions = {|
+    display_root_level_toolbar?: mixed,
+    get_indicators?: mixed,
+    showHeader?: boolean,
+    skin?: Skin,
+|};
```

Comment:
i think its a bit confusing to type the options exlicitly because different adapter will implement different options. also third party adapters might implement options that are not part of the type 😕 
so i think i would keep using `{[key: string]: mixed}`

```suggestion
export type AdapterOptions = {[key: string]: mixed};
```